src/Stratis.Bitcoin.Features.MemoryPool/MempoolOrphans.cs

```diff
@@ -259,15 +265,18 @@ public class OrphanTx
         /// <returns>Whether the transaction was added to orphans.</returns>
         public async Task<bool> ProcessesOrphansMissingInputsAsync(INetworkPeer from, Transaction tx)
         {
+            this.mempoolLogger.LogTrace("{0}:'{1}', {2}:'{3}'", nameof(from), from?.RemoteSocketEndpoint, nameof(tx), tx?.GetHash());
             // It may be the case that the orphans parents have all been rejected
             var rejectedParents = await this.MempoolLock.ReadAsync(() =>
             {
+                this.mempoolLogger.LogTrace("Executing task to query if orphans parents have all been rejected.");
```

Comment:
Please add exit log with the return value

Reply:
Not sure this logging makes any sense 😁 I think it's safely covered by the block just below it, so just got rid of it